modules/dynamic-plugins/ref-community-plugins.adoc

```diff
@@ -3,7 +3,14 @@
 
 = Community plugins
 
-{product-short} includes the following 43 community plugins:
+[IMPORTANT]
+====
+{product} ({product-very-short}) includes a select number of community-supported plugins, available for customers to enable and configure. These community plugins are exported by {company-name} as dynamic plugins, and are provided with support scoped per https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Scope] terms. 
```

Comment:
This link outlines the scope for Tech Preview features. Is the scope similar for community plugins as well?

Reply:
Similar wording is used in the [Red Hat Developer Hub Support for bundled community plugins](https://access.redhat.com/policy/developerhub-support-policy) statement, albeit as follows:
_These **included** community plugins **are augmented by Red Hat to be dynamic plugin capable**, and are provided with support scoped per Technical Preview terms._

@reestr Should we update this reference to TP to match how it is phrased in the _Red Hat Developer Hub Support for bundled community plugins?

Reply:
@Gerry-Forde - yes, and please link to the RHDH support page for community plugins - that is where an expanded version of what support means to the customer can be found.

Reply:
Updated statement as ^^^ and linked to the RHDH support policy page for community support statement at https://access.redhat.com/policy/developerhub-support-policy